### ISSUE TITLE
Issue with luisRecognizer.js

### ISSUE THREAD

**Issue by Kperroch** [Microsoft/botbuilder-js#522]
The luisRecognizer does not take into account the optional parameters in the query string.

the staging parameter for instance.

The correction can be : 
```
recognize(context) {
        const cached = context.turnState.get(this.cacheKey);
        if (!cached) {
            const utterance = context.activity.text || '';
            return this.luisClient.prediction.resolve(this.application.applicationId, utterance, {
                timezoneOffset: this.options.timezoneOffset,
                verbose: this.options.includeAllIntents,
                log: this.options.log,
		staging: this.options.staging,
                customHeaders: { 'Ocp-Apim-Subscription-Key': this.application.endpointKey }
            })
```

**Comment by JasonSowers** [comment]
@Kperroch would you like to submit a pull request for this change so we can review it?

**Comment by stevengum** [comment]
This was fixed with #530, thanks for reporting the bug.
